LiteFarmOrg/LiteFarm
# Lf 2503: Soil water potential reading component

To Test,

1. Add beta dump to the local database
2. go to the farm map page
3. click on a sensor 
4. on the readings page check the sensor readings for the soli water potential 

<img width="979" alt="Screen Shot 2022-12-20 at 10 43 41 AM" src="https://user-images.githubusercontent.com/20675885/208800795-948b17be-716d-4f7d-a541-1d08f9060f66.png">

<img width="1011" alt="Screen Shot 2022-12-20 at 10 43 51 AM" src="https://user-images.githubusercontent.com/20675885/208800797-96202fca-6ecf-45fd-8eca-77dc3bb36cfd.png">

<img width="795" alt="Screen Shot 2022-12-20 at 5 44 11 PM" src="https://user-images.githubusercontent.com/20675885/208800872-0626d63e-6e3a-42a8-9b53-afedf0966e23.png">
